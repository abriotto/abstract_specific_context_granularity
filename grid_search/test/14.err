  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  7%|#############2                                                                                                                                                                              | 7/99 [00:00<00:01, 63.58it/s] 17%|################################1                                                                                                                                                          | 17/99 [00:00<00:01, 79.54it/s] 25%|###############################################2                                                                                                                                           | 25/99 [00:00<00:01, 63.83it/s] 32%|############################################################4                                                                                                                              | 32/99 [00:00<00:01, 60.60it/s] 39%|#########################################################################6                                                                                                                 | 39/99 [00:00<00:01, 55.33it/s] 45%|#####################################################################################                                                                                                      | 45/99 [00:00<00:01, 52.60it/s] 54%|####################################################################################################1                                                                                      | 53/99 [00:00<00:00, 58.36it/s] 61%|#################################################################################################################3                                                                         | 60/99 [00:01<00:00, 59.88it/s] 68%|##############################################################################################################################5                                                            | 67/99 [00:01<00:00, 58.56it/s] 74%|#########################################################################################################################################8                                                 | 73/99 [00:01<00:00, 57.38it/s] 81%|#######################################################################################################################################################1                                   | 80/99 [00:01<00:00, 56.76it/s] 89%|######################################################################################################################################################################2                    | 88/99 [00:01<00:00, 61.32it/s] 96%|###################################################################################################################################################################################4       | 95/99 [00:01<00:00, 62.01it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 60.77it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 20%|#####################################6                                                                                                                                                      | 5/25 [00:00<00:00, 39.54it/s] 56%|########################################################################################################7                                                                                  | 14/25 [00:00<00:00, 58.32it/s] 88%|####################################################################################################################################################################5                      | 22/25 [00:00<00:00, 60.77it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 56.67it/s]
