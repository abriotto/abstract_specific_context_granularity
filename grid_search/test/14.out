# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6980118155479431, "acc": 0.512212872505188, "length": 3.583493232727051, "mode": "train", "epoch": 1}
{"loss": 0.6944150924682617, "acc": 0.5046555399894714, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6929402351379395, "acc": 0.5315295457839966, "length": 2.8476836681365967, "mode": "train", "epoch": 2}
{"loss": 0.6910635232925415, "acc": 0.5037244558334351, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6827921867370605, "acc": 0.5720438957214355, "length": 2.408416509628296, "mode": "train", "epoch": 3}
{"loss": 0.6686114072799683, "acc": 0.6226034164428711, "length": 1.6703910827636719, "mode": "test", "epoch": 3}
{"loss": 0.6451359391212463, "acc": 0.6611431837081909, "length": 2.473729372024536, "mode": "train", "epoch": 4}
{"loss": 0.6142946481704712, "acc": 0.6890421509742737, "length": 2.0726256370544434, "mode": "test", "epoch": 4}
{"loss": 0.5868435502052307, "acc": 0.7130517959594727, "length": 2.900409460067749, "mode": "train", "epoch": 5}
{"loss": 0.5513152480125427, "acc": 0.7231804132461548, "length": 3.5474860668182373, "mode": "test", "epoch": 5}
{"loss": 0.5333319306373596, "acc": 0.7444506287574768, "length": 3.56028413772583, "mode": "train", "epoch": 6}
{"loss": 0.5168778896331787, "acc": 0.7655883431434631, "length": 3.7374300956726074, "mode": "test", "epoch": 6}
{"loss": 0.4741450548171997, "acc": 0.7849990129470825, "length": 3.723353385925293, "mode": "train", "epoch": 7}
{"loss": 0.5022910833358765, "acc": 0.7858763337135315, "length": 3.9888267517089844, "mode": "test", "epoch": 7}
{"loss": 0.4455544054508209, "acc": 0.8001460433006287, "length": 3.8593575954437256, "mode": "train", "epoch": 8}
{"loss": 0.5070580244064331, "acc": 0.7569833397865295, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.4346012473106384, "acc": 0.8050528168678284, "length": 3.936966896057129, "mode": "train", "epoch": 9}
{"loss": 0.48918819427490234, "acc": 0.7625698447227478, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.41667643189430237, "acc": 0.817288875579834, "length": 3.948838710784912, "mode": "train", "epoch": 10}
{"loss": 0.4713665843009949, "acc": 0.8007449507713318, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.3966274857521057, "acc": 0.8256441950798035, "length": 3.9679486751556396, "mode": "train", "epoch": 11}
{"loss": 0.4869152307510376, "acc": 0.7765362858772278, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.3767743408679962, "acc": 0.8415112495422363, "length": 3.978896141052246, "mode": "train", "epoch": 12}
{"loss": 0.48138493299484253, "acc": 0.7979516386985779, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.37639808654785156, "acc": 0.8471478223800659, "length": 3.985729932785034, "mode": "train", "epoch": 13}
{"loss": 0.4646011292934418, "acc": 0.8119181394577026, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.35963204503059387, "acc": 0.850584089756012, "length": 3.9842660427093506, "mode": "train", "epoch": 14}
{"loss": 0.4486677646636963, "acc": 0.8230913281440735, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.34417417645454407, "acc": 0.8606296181678772, "length": 3.99168062210083, "mode": "train", "epoch": 15}
{"loss": 0.44089123606681824, "acc": 0.8268157243728638, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.3386252224445343, "acc": 0.8597270846366882, "length": 3.9943487644195557, "mode": "train", "epoch": 16}
{"loss": 0.4266149401664734, "acc": 0.8268155455589294, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.33057597279548645, "acc": 0.8641025424003601, "length": 3.9945731163024902, "mode": "train", "epoch": 17}
{"loss": 0.44787704944610596, "acc": 0.8212291598320007, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.3256596326828003, "acc": 0.8684611320495605, "length": 3.9959723949432373, "mode": "train", "epoch": 18}
{"loss": 0.4275538921356201, "acc": 0.8240223526954651, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.3204137980937958, "acc": 0.8703532218933105, "length": 3.9966275691986084, "mode": "train", "epoch": 19}
{"loss": 0.41577479243278503, "acc": 0.8370577692985535, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.30721738934516907, "acc": 0.878121018409729, "length": 3.9962105751037598, "mode": "train", "epoch": 20}
{"loss": 0.41328972578048706, "acc": 0.8296089172363281, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.3105173110961914, "acc": 0.8697443008422852, "length": 3.9976205825805664, "mode": "train", "epoch": 21}
{"loss": 0.40020620822906494, "acc": 0.8472997546195984, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.2956364154815674, "acc": 0.8812445402145386, "length": 3.9970414638519287, "mode": "train", "epoch": 22}
{"loss": 0.4049142301082611, "acc": 0.8472998142242432, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.2879336476325989, "acc": 0.8881291151046753, "length": 3.9980757236480713, "mode": "train", "epoch": 23}
{"loss": 0.3896569013595581, "acc": 0.852886438369751, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.2870458960533142, "acc": 0.8837722539901733, "length": 3.9981391429901123, "mode": "train", "epoch": 24}
{"loss": 0.4059174656867981, "acc": 0.8482309579849243, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.2862735688686371, "acc": 0.8797183632850647, "length": 3.998223066329956, "mode": "train", "epoch": 25}
{"loss": 0.3857603967189789, "acc": 0.854748547077179, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.27532514929771423, "acc": 0.8903326988220215, "length": 3.999403953552246, "mode": "train", "epoch": 26}
{"loss": 0.3966275155544281, "acc": 0.8463686108589172, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.27230343222618103, "acc": 0.8847214579582214, "length": 3.9987926483154297, "mode": "train", "epoch": 27}
{"loss": 0.3857671618461609, "acc": 0.8398510813713074, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.2693119943141937, "acc": 0.8915700912475586, "length": 3.999223232269287, "mode": "train", "epoch": 28}
{"loss": 0.3838258385658264, "acc": 0.8556798100471497, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.2572363615036011, "acc": 0.9021638631820679, "length": 3.9991602897644043, "mode": "train", "epoch": 29}
{"loss": 0.36977311968803406, "acc": 0.8575419783592224, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.2679844796657562, "acc": 0.8853501677513123, "length": 3.999572515487671, "mode": "train", "epoch": 30}
{"loss": 0.36991509795188904, "acc": 0.8594042658805847, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.28008800745010376, "acc": 0.8940714001655579, "length": 3.9992332458496094, "mode": "train", "epoch": 31}
{"loss": 0.3794691562652588, "acc": 0.8575419783592224, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.2604714035987854, "acc": 0.8979061841964722, "length": 3.9940366744995117, "mode": "train", "epoch": 32}
{"loss": 0.3846326470375061, "acc": 0.852886438369751, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.24401740729808807, "acc": 0.9009233713150024, "length": 3.9995830059051514, "mode": "train", "epoch": 33}
{"loss": 0.36601191759109497, "acc": 0.8612664341926575, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.24663299322128296, "acc": 0.9024790525436401, "length": 3.9987893104553223, "mode": "train", "epoch": 34}
{"loss": 0.3792601227760315, "acc": 0.8575419783592224, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.2417711317539215, "acc": 0.9065393209457397, "length": 3.999913215637207, "mode": "train", "epoch": 35}
{"loss": 0.3700527548789978, "acc": 0.8612664341926575, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.23624655604362488, "acc": 0.9068450927734375, "length": 3.9997079372406006, "mode": "train", "epoch": 36}
{"loss": 0.35862576961517334, "acc": 0.8519554138183594, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.2406768500804901, "acc": 0.9034246802330017, "length": 3.9998621940612793, "mode": "train", "epoch": 37}
{"loss": 0.3696865439414978, "acc": 0.8631286025047302, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.24000228941440582, "acc": 0.9021615386009216, "length": 3.999422311782837, "mode": "train", "epoch": 38}
{"loss": 0.3735898733139038, "acc": 0.8538175821304321, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.2362903505563736, "acc": 0.904340386390686, "length": 3.9994773864746094, "mode": "train", "epoch": 39}
{"loss": 0.3524901270866394, "acc": 0.8594041466712952, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.2406676858663559, "acc": 0.9043610095977783, "length": 3.9999845027923584, "mode": "train", "epoch": 40}
{"loss": 0.3896670341491699, "acc": 0.8556798100471497, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.2368583381175995, "acc": 0.9074744582176208, "length": 3.9999420642852783, "mode": "train", "epoch": 41}
{"loss": 0.3694370687007904, "acc": 0.8575419783592224, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.2272171974182129, "acc": 0.9056047797203064, "length": 3.999931573867798, "mode": "train", "epoch": 42}
{"loss": 0.37147316336631775, "acc": 0.8659219741821289, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.23199819028377533, "acc": 0.9118337631225586, "length": 3.9997434616088867, "mode": "train", "epoch": 43}
{"loss": 0.36092135310173035, "acc": 0.8687151670455933, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.22859340906143188, "acc": 0.9124607443809509, "length": 3.9999845027923584, "mode": "train", "epoch": 44}
{"loss": 0.3797348737716675, "acc": 0.8584731221199036, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.2327394038438797, "acc": 0.9076708555221558, "length": 3.9971835613250732, "mode": "train", "epoch": 45}
{"loss": 0.39758801460266113, "acc": 0.8435754776000977, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.22057144343852997, "acc": 0.9115260243415833, "length": 3.999986410140991, "mode": "train", "epoch": 46}
{"loss": 0.3686341941356659, "acc": 0.8640596866607666, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.21308289468288422, "acc": 0.9193145632743835, "length": 3.999998092651367, "mode": "train", "epoch": 47}
{"loss": 0.38335031270980835, "acc": 0.852886438369751, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.21284735202789307, "acc": 0.9180682897567749, "length": 3.999988555908203, "mode": "train", "epoch": 48}
{"loss": 0.35478946566581726, "acc": 0.8668529987335205, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.23488254845142365, "acc": 0.9068537354469299, "length": 3.999992609024048, "mode": "train", "epoch": 49}
{"loss": 0.37307724356651306, "acc": 0.8594040870666504, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.2419101744890213, "acc": 0.9068292379379272, "length": 3.9993338584899902, "mode": "train", "epoch": 50}
{"loss": 0.3557833433151245, "acc": 0.8640596270561218, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.21454410254955292, "acc": 0.9152628779411316, "length": 3.9999146461486816, "mode": "train", "epoch": 51}
{"loss": 0.3560248613357544, "acc": 0.8631284832954407, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.21675702929496765, "acc": 0.9165106415748596, "length": 3.999992609024048, "mode": "train", "epoch": 52}
{"loss": 0.3993460536003113, "acc": 0.8556798100471497, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.22189109027385712, "acc": 0.9161984324455261, "length": 3.9999754428863525, "mode": "train", "epoch": 53}
{"loss": 0.3451407849788666, "acc": 0.8659217953681946, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.21362456679344177, "acc": 0.9165106415748596, "length": 3.9999959468841553, "mode": "train", "epoch": 54}
{"loss": 0.36635875701904297, "acc": 0.8612663149833679, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.2041950225830078, "acc": 0.9214495420455933, "length": 3.998533248901367, "mode": "train", "epoch": 55}
{"loss": 0.35360443592071533, "acc": 0.8594041466712952, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.2045259028673172, "acc": 0.9246103167533875, "length": 3.9999914169311523, "mode": "train", "epoch": 56}
{"loss": 0.35343098640441895, "acc": 0.8668529987335205, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.2173357456922531, "acc": 0.918065071105957, "length": 3.9999032020568848, "mode": "train", "epoch": 57}
{"loss": 0.3622865378856659, "acc": 0.8798883557319641, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.20905958116054535, "acc": 0.923052966594696, "length": 3.999997615814209, "mode": "train", "epoch": 58}
{"loss": 0.36038440465927124, "acc": 0.8743017315864563, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.2045944780111313, "acc": 0.9252331852912903, "length": 3.9999871253967285, "mode": "train", "epoch": 59}
{"loss": 0.35290640592575073, "acc": 0.8761639595031738, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.20530244708061218, "acc": 0.9230518937110901, "length": 3.999894142150879, "mode": "train", "epoch": 60}
{"loss": 0.35613903403282166, "acc": 0.8649906516075134, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.20259563624858856, "acc": 0.9249220490455627, "length": 3.9999985694885254, "mode": "train", "epoch": 61}
{"loss": 0.36360400915145874, "acc": 0.8845438361167908, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.21284838020801544, "acc": 0.9202489256858826, "length": 3.9999899864196777, "mode": "train", "epoch": 62}
{"loss": 0.4260287284851074, "acc": 0.8398510813713074, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.23153412342071533, "acc": 0.9080996513366699, "length": 3.9999985694885254, "mode": "train", "epoch": 63}
{"loss": 0.3738176226615906, "acc": 0.8547486066818237, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.20702490210533142, "acc": 0.9242992401123047, "length": 3.9999125003814697, "mode": "train", "epoch": 64}
{"loss": 0.3557688593864441, "acc": 0.870577335357666, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.20046907663345337, "acc": 0.9239862561225891, "length": 3.999969959259033, "mode": "train", "epoch": 65}
{"loss": 0.3533444404602051, "acc": 0.8724395036697388, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.1987030953168869, "acc": 0.9283478260040283, "length": 3.9999771118164062, "mode": "train", "epoch": 66}
{"loss": 0.35375604033470154, "acc": 0.8724395632743835, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.20026864111423492, "acc": 0.9168114066123962, "length": 3.9996469020843506, "mode": "train", "epoch": 67}
{"loss": 0.3579554557800293, "acc": 0.8659219741821289, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.19399452209472656, "acc": 0.9308393001556396, "length": 3.999948501586914, "mode": "train", "epoch": 68}
{"loss": 0.3449011743068695, "acc": 0.8826815485954285, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.18976210057735443, "acc": 0.932705283164978, "length": 3.9998626708984375, "mode": "train", "epoch": 69}
{"loss": 0.3668389320373535, "acc": 0.8594041466712952, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.18474000692367554, "acc": 0.9305294156074524, "length": 3.9999985694885254, "mode": "train", "epoch": 70}
{"loss": 0.3599456250667572, "acc": 0.8594041466712952, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.19279389083385468, "acc": 0.9280375242233276, "length": 3.999998092651367, "mode": "train", "epoch": 71}
{"loss": 0.3627622723579407, "acc": 0.8696463108062744, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.18187744915485382, "acc": 0.9323651790618896, "length": 3.9985480308532715, "mode": "train", "epoch": 72}
{"loss": 0.35764166712760925, "acc": 0.8696463108062744, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.18755365908145905, "acc": 0.9289716482162476, "length": 3.9999899864196777, "mode": "train", "epoch": 73}
{"loss": 0.33273667097091675, "acc": 0.8882681727409363, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.17941734194755554, "acc": 0.9330212473869324, "length": 3.9999871253967285, "mode": "train", "epoch": 74}
{"loss": 0.38539713621139526, "acc": 0.8454376459121704, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.1915724128484726, "acc": 0.9295949339866638, "length": 3.999998092651367, "mode": "train", "epoch": 75}
{"loss": 0.39026662707328796, "acc": 0.8621973395347595, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.19359415769577026, "acc": 0.9292834401130676, "length": 3.999998092651367, "mode": "train", "epoch": 76}
{"loss": 0.3691926598548889, "acc": 0.8761638402938843, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.16796189546585083, "acc": 0.9408098459243774, "length": 3.9999945163726807, "mode": "train", "epoch": 77}
{"loss": 0.36583107709884644, "acc": 0.8770949840545654, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.1829114705324173, "acc": 0.9352022409439087, "length": 3.999983549118042, "mode": "train", "epoch": 78}
{"loss": 0.3854316174983978, "acc": 0.8640596270561218, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.18141253292560577, "acc": 0.931462824344635, "length": 3.999951124191284, "mode": "train", "epoch": 79}
{"loss": 0.34408116340637207, "acc": 0.8743017315864563, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.17478017508983612, "acc": 0.9364484548568726, "length": 3.99999737739563, "mode": "train", "epoch": 80}
{"loss": 0.3463171124458313, "acc": 0.875232994556427, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.17242057621479034, "acc": 0.9358215928077698, "length": 3.999910593032837, "mode": "train", "epoch": 81}
{"loss": 0.357761025428772, "acc": 0.8752328753471375, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.16319137811660767, "acc": 0.9401755928993225, "length": 3.99981689453125, "mode": "train", "epoch": 82}
{"loss": 0.35153812170028687, "acc": 0.882681667804718, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.15809738636016846, "acc": 0.942368745803833, "length": 3.999891757965088, "mode": "train", "epoch": 83}
{"loss": 0.3580324649810791, "acc": 0.864990770816803, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.16731052100658417, "acc": 0.9373821020126343, "length": 3.9999799728393555, "mode": "train", "epoch": 84}
{"loss": 0.3576951026916504, "acc": 0.8770950436592102, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.16485217213630676, "acc": 0.9367600679397583, "length": 3.9999959468841553, "mode": "train", "epoch": 85}
{"loss": 0.36151251196861267, "acc": 0.8687151670455933, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.16015218198299408, "acc": 0.9383177161216736, "length": 3.999998092651367, "mode": "train", "epoch": 86}
{"loss": 0.35369592905044556, "acc": 0.870577335357666, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.17659416794776917, "acc": 0.9370715618133545, "length": 3.9999921321868896, "mode": "train", "epoch": 87}
{"loss": 0.33162468671798706, "acc": 0.8845437169075012, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.1586667150259018, "acc": 0.9433020949363708, "length": 3.999999523162842, "mode": "train", "epoch": 88}
{"loss": 0.34804725646972656, "acc": 0.8817505836486816, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.16345429420471191, "acc": 0.9436129927635193, "length": 3.9999871253967285, "mode": "train", "epoch": 89}
{"loss": 0.3565713167190552, "acc": 0.8668529391288757, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.17176155745983124, "acc": 0.9367599487304688, "length": 3.999992609024048, "mode": "train", "epoch": 90}
{"loss": 0.3224106431007385, "acc": 0.8836126923561096, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.1657184660434723, "acc": 0.9414323568344116, "length": 3.999990463256836, "mode": "train", "epoch": 91}
{"loss": 0.3471263349056244, "acc": 0.8780261278152466, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.16080538928508759, "acc": 0.943301796913147, "length": 3.999988079071045, "mode": "train", "epoch": 92}
{"loss": 0.33326455950737, "acc": 0.8854749202728271, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.149590402841568, "acc": 0.9448594450950623, "length": 3.9999821186065674, "mode": "train", "epoch": 93}
{"loss": 0.33992260694503784, "acc": 0.8808194994926453, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.15612898766994476, "acc": 0.9432974457740784, "length": 3.9999237060546875, "mode": "train", "epoch": 94}
{"loss": 0.36212795972824097, "acc": 0.8724395632743835, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.15243662893772125, "acc": 0.9464174509048462, "length": 3.9999985694885254, "mode": "train", "epoch": 95}
{"loss": 0.372332900762558, "acc": 0.872439444065094, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.16056033968925476, "acc": 0.9395638704299927, "length": 3.9999990463256836, "mode": "train", "epoch": 96}
{"loss": 0.36937522888183594, "acc": 0.8733706474304199, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.20049996674060822, "acc": 0.927395224571228, "length": 3.999687910079956, "mode": "train", "epoch": 97}
{"loss": 0.421377956867218, "acc": 0.8510242700576782, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.22641503810882568, "acc": 0.9174270629882812, "length": 3.9997479915618896, "mode": "train", "epoch": 98}
{"loss": 0.35235878825187683, "acc": 0.8705773949623108, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.17074380815029144, "acc": 0.93706876039505, "length": 3.9999465942382812, "mode": "train", "epoch": 99}
{"loss": 0.33642578125, "acc": 0.8808194994926453, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.1641763150691986, "acc": 0.9404975771903992, "length": 3.999964952468872, "mode": "train", "epoch": 100}
{"loss": 0.3794121742248535, "acc": 0.8696463108062744, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.7419354915618896
